packages/litexa-html/README.md

```diff
@@ -55,7 +167,7 @@ launch
   END
 ```
 
-This will add a substring like `<mark name="screen:blue"/>` to your response's speech. 
+This will add a substring like `<mark name="screen:blue"/>` to your response's speech.
 This is useful for your HTML runtime in that, when it receives the Alexa response,
 the marks can be used to trigger HTML events that interleave with SSML playback.
```

Comment:
This needed more explanation - particularly that using marks on the HTML side requires:
1.) a litexa postprocessor that moves (or adds, whatever you want to do) SSML output to HTML message input
2.) a transformer (explained in Web API docs I assume) to pull the marks out from the message data and trigger events corresponding to the mark tags.